src/Adapter/CustomerService/CommandHandler/ReplyToCustomerThreadHandler.php

```diff
@@ -138,7 +138,7 @@ private function sendReplyEmail(CustomerThread $customerThread, CustomerMessage
         $contact = new Contact((int) $customerThread->id_contact, (int) $customerThread->id_lang);
 
         if (Validate::isLoadedObject($contact)) {
-            $fromName = $contact->name;
+            $fromName = is_array($contact->name) ? reset($contact->name) : $contact->name;
```

Comment:
is PHPStan forcing this? it should never been an array if we have provided language id to $contact object 🤔

Reply:
Yes, because $name property can be an array or a string. It's the problem with mixed type property.